charts/spire/charts/spire-server/templates/configmap.yaml

```
@@ -1,126 +1,110 @@
-data:
-  server.conf: |
-    server {
-      bind_address = "0.0.0.0"
-      bind_port = "8081"
-      trust_domain = {{ include "spire-server.trust-domain" . | quote }}
-      data_dir = "/run/spire/data"
-      log_level = {{ .Values.logLevel | quote }}
+server:
+  bind_address: "0.0.0.0"
+  bind_port: "8081"
+  trust_domain: {{ include "spire-server.trust-domain" . | quote }}
+  data_dir: "/run/spire/data"
+  log_level: {{ .Values.logLevel | quote }}
+  jwt_issuer: {{ .Values.jwtIssuer | quote }}
 
-      jwt_issuer = {{ .Values.jwtIssuer | quote }}
+  ca_key_type: {{ .Values.caKeyType | quote }}
+  ca_ttl: {{ .Values.caTTL | quote }}
 
-      ca_key_type = {{ .Values.caKeyType | quote }}
-      ca_ttl = {{ .Values.caTTL | quote }}
+  default_x509_svid_ttl: {{ .Values.defaultX509SvidTTL | quote }}
+  default_jwt_svid_ttl: {{ .Values.defaultJwtSvidTTL | quote }}
 
-      default_x509_svid_ttl = {{ .Values.defaultX509SvidTTL | quote }}
-      default_jwt_svid_ttl = {{ .Values.defaultJwtSvidTTL | quote }}
-
-      ca_subject = {
-        {{- with .Values.ca_subject }}
-        country = [{{ .country | quote }}],
-        organization = [{{ .organization | quote }}],
-        common_name = {{ .common_name | quote }},
-        {{- end }}
-      }
+  ca_subject:
+    {{- with .Values.ca_subject }}
+    - country: [{{ .country | quote }}]
+      organization: [{{ .organization | quote }}]
+      common_name: {{ .common_name | quote }}
+    {{- end }}
 
-      {{- with .Values.federation }}
-      {{- if eq (.enabled | toString) "true" }}
-      federation {
-        bundle_endpoint {
-          address = "{{ .bundleEndpoint.address }}"
-          port = {{ .bundleEndpoint.port }}
-        }
-      }
-      {{- end }}
-      {{- end }}
-    }
+  {{- with .Values.federation }}
+  {{- if eq (.enabled | toString) "true" }}
+  federation:
+    bundle_endpoint:
+      - {{ .bundleEndpoint | toYaml | nindent 8 }}
+  {{- end }}
+  {{- end }}
 
-    plugins {
-      DataStore "sql" {
-        plugin_data {
-          database_type = "sqlite3"
-          connection_string = "/run/spire/data/datastore.sqlite3"
-        }
-      }
+plugins:
+  DataStore:
+    - sql:
+        plugin_data:
+          database_type: "sqlite3"
+          connection_string: "/run/spire/data/datastore.sqlite3"
 
-      {{- with .Values.nodeAttestor.k8sPsat }}
-      {{- if eq (.enabled | toString) "true" }}
-      NodeAttestor "k8s_psat" {
-        plugin_data {
-          clusters = {
-            {{ include "spire-server.cluster-name" $root | quote }} = {
-              service_account_allow_list = {{ include "spire-server.serviceAccountAllowedList" $root | trim }}
-            }
-          }
-        }
-      }
-      {{- end }}
-      {{- end }}
+  {{- with .Values.nodeAttestor.k8sPsat }}
+  {{- if eq (.enabled | toString) "true" }}
+  NodeAttestor:
+    - k8s_psat:
+        plugin_data:
+          clusters:
+            {{ $root.Values.clusterName }}:
```

One other merge conflict seems to have made it in:

```suggestion
            {{ include "spire-server.cluster-name" $root }}:
```